types/react-redux/index.d.ts

```diff
@@ -445,3 +466,91 @@ export const ReactReduxContext: Context<ReactReduxContextValue>;
  * multiple actions dispatched outside of React only result in a single render update.
  */
 export function batch(cb: () => void): void;
+
+/**
+ * Compares two arbitrary values for shallow equality. Object values are compared based on their keys, i.e. they must
+ * have the same keys and for each key the value must be equal according to the `Object.is()` algorithm. Non-object
```

Comment:
We're not actually using `Object.is()` or its equivalent right now - it's just a straight-up reference `===`.

Reply:
@markerikson Are you maybe confusing this with the `useSelector` hook? The `shallowEqual` function is using `Object.is()` as you can see [here](https://github.com/reduxjs/react-redux/blob/v7-hooks-alpha/src/utils/shallowEqual.js).

Reply:
Whoops, yeah :)